docs/open-api/lineage.yaml

```diff
@@ -0,0 +1,281 @@
+        schemaURL:
+          description: The JSON Pointer (https://tools.ietf.org/html/rfc6901) URL to the corresponding version of the schema definition for this RunEvent
+          type: string
+          format: uri
+          example: https://openlineage.io/spec/0-0-1/OpenLineage.json
+      required:
+        - eventTime
+        - producer
+        - schemaURL
+    BaseFacet:
+      description: all fields of the base facet are prefixed with _ to avoid name conflicts in facets
+      type: object
+      properties:
+        _producer:
+          description: URI identifying the producer of this metadata. For example this could be a git url with a given tag or sha
+          type: string
+          format: uri
+          example: https://github.com/OpenLineage/OpenLineage/blob/v1-0-0/client
+        _schemaURL:
+          description: The JSON Pointer (https://tools.ietf.org/html/rfc6901) URL to the corresponding version of the schema definition for this facet
+          type: string
+          format: uri
+          example: https://openlineage.io/spec/1-0-2/OpenLineage.json#/$defs/BaseFacet
+      additionalProperties: true
+      required:
+        - _producer
+        - _schemaURL
+    RunFacet:
+      description: A Run Facet
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/BaseFacet'
+    Run:
+      type: object
+      properties:
+        runId:
+          description: The globally unique ID of the run associated with the job.
+          type: string
+          format: uuid
+        facets:
+          description: The run facets.
+          type: object
+          anyOf:
+            - type: object
+              additionalProperties:
+                $ref: '#/components/schemas/RunFacet'
+      required:
+        - runId
+    JobFacet:
+      description: A Job Facet
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/BaseFacet'
+        - type: object
+          properties:
+            _deleted:
+              description: set to true to delete a facet
+              type: boolean
+    Job:
+      type: object
+      properties:
+        namespace:
+          description: The namespace containing that job
+          type: string
+          example: my-scheduler-namespace
+        name:
+          description: The unique name for that job within that namespace
+          type: string
+          example: myjob.mytask
+        facets:
+          description: The job facets.
+          type: object
+          anyOf:
+            - type: object
+              additionalProperties:
+                $ref: '#/components/schemas/JobFacet'
+      required:
+        - namespace
+        - name
+    DatasetFacet:
+      description: A Dataset Facet
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/BaseFacet'
+        - type: object
+          properties:
+            _deleted:
+              description: set to true to delete a facet
+              type: boolean
```

Comment:
This "hierarchy" looks like an over design to me.
Can we simply merge the `_deleted` property to `BaseFacet` and mark it optional?